datalake/backend/tests/test_dataset_versions_endpoint.py

```diff
@@ -0,0 +1,104 @@
+"""
+Dataset Versions endpoint Lambda function tests. Working Data Lake AWS environment is
+required (run '$ cdk deploy' before running tests).
```

Comment:
I think

```
`cdk deploy`
```

would be a more common way to write `'$ cdk deploy'`. The `$` prompt is usually only included with a command when showing a session recording, including the output of commands.

In any case, I think these commands should be `@mark.infrastructure` if they actually require a deployed system. We should probably document that mark in README.md in a separate PR rather than alongside each infrastructure test case.